weave/trace/op.py

```diff
@@ -140,6 +149,17 @@ async def _run_async() -> Coroutine[Any, Any, Any]:
         else:
             return on_output(res)
 
+    def call(self, *args: Any, **kwargs: Any) -> "Call":
+        client = client_context.weave_client.require_weave_client()
+        run = self._create_run(*args, **kwargs)
+        self._execute_run(run, *args, **kwargs)
+        return self._retry_fetch_call(client, run.id)
+
+    @staticmethod
+    @retry
```

Comment:
I think this will retry forever with no backoff? We should probably limit it somehow.